modules/gltf/src/lib/parse-gltf.js

```diff
@@ -174,9 +173,9 @@ async function fetchImages(gltf, options) {
 }
 
 // Asynchronously fetches and parses one image, store in images array outside of json
-async function fetchAndParseLinkedImage(gltf, image, i, options) {
-  const fetch = options.fetch || window.fetch;
-  assert(fetch);
+async function fetchAndParseLinkedImage(gltf, image, i, options, context) {
+  // const fetch = options.fetch || window.fetch;
+  // assert(fetch);
```

Comment:
why comment out these lines?

Reply:
This should be done with `context`. Not used right now